### DISCUSSION TITLE
How to catch error

### DISCUSSION THREAD

**Discussion by JSDUNIYA** [orval-labs/orval#374] [Q&A] [answered]
How to handle error after using query from orval generated code ?

Example:

useMypets({query()});  // Here if my API fails how to handle ?

**Comment by anymaniax** [comment, marked as answer]
Like with react query using the onError function

**Reply by saulhoward** [reply]
This is no longer recommended and will be removed: https://tkdodo.eu/blog/breaking-react-querys-api-on-purpose

**Reply by anymaniax** [reply]
@saulhoward for useQuery but not for useMutation

**Reply by saulhoward** [reply]
Ah... thank you! I'm in the middle of debugging an issue with this and that's a great help :D 

BTW -- thanks for an awesome library!
